app/controllers/applications/process_controller.rb

```diff
@@ -17,15 +18,35 @@ def personal_information_save
       end
     end
 
+    def summary
+      @result = Views::Applikation::Result.new(application)
+      @overview = Views::ApplicationOverview.new(application)
+    end
+
+    def summary_save
```

Comment:
I would remove this action for now, until we need it. I think the `Complete processing` link should direct straight to the confirmation page for now, because it's just an extra redirect at this stage.